src/Sylius/Bundle/AdminBundle/Twig/Component/Dashboard/NewCustomersComponent.php

```diff
@@ -34,7 +34,7 @@ public function __construct(
     /**
      * @return array<CustomerInterface>
      */
-    #[ExposeInTemplate]
+    #[ExposeInTemplate('new_customers')]
```

Comment:
we should do the same with `Sylius\Bundle\AdminBundle\Twig\Component\Order\AddressHistoryComponent`, `Sylius\Bundle\AdminBundle\Twig\Component\Taxon\TreeComponent`, `Sylius\Bundle\AdminBundle\Twig\Component\Product\ProductAttributeAutocompleteComponent`, `Sylius\Bundle\AdminBundle\Twig\Component\Shared\Navbar\UserDropdownComponent` to get rid of the rest camel case variables in twigs 🤔